cloudflare/resource_cloudflare_filter.go

```diff
@@ -40,6 +40,9 @@ func resourceCloudflareFilter() *schema.Resource {
 			"expression": {
 				Type:     schema.TypeString,
 				Required: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return strings.Trim(new, "\n\t ") == old
```

Comment:
I think [`TrimSpace`](https://golang.org/pkg/strings/#TrimSpace) is a better option here as it caters for all types of whitespace (as defined by [`unicode.IsSpace`](https://github.com/golang/go/blob/master/src/unicode/graphic.go#L126-L136)) instead of us maintaining a dedicated list.

```suggestion
					return strings.TrimSpace(new) == old
```